### DISCUSSION TITLE
Error! Project not found - while deploying new project

### DISCUSSION THREAD

**Discussion by dimad3** [vercel/community#1205] [Help]
### Summary

I try to deploy new Angular project by running:
`now`
in the terminal from this project directory. Unfortunately this error occurs:
```
Error! Project not found
? What’s the name of your existing project?
```

How to fix it, please!

**Comment by amyegan** [comment]
Thanks for reaching out here @dimad3! The [`now`](https://www.npmjs.com/package/now) package is deprecated and you should switch to using [`vercel`](https://www.npmjs.com/package/vercel) instead. If you try to [deploy your Angular app using Vercel CLI](https://vercel.com/guides/deploying-angular-with-vercel#vercel-cli), do you get the same error?

**Reply by dimad3** [reply]
Yes I have the same error

**Reply by amyegan** [reply]
Do you have a public repository or [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share? This sounds like a tricky problem and I need to get a closer look 👀
